src/processor/operator/hash_join/join_hash_table.cpp

```diff
@@ -184,16 +179,8 @@ sel_t JoinHashTable::matchUnFlatKey(ValueVector* keyVector, uint8_t** probedTupl
     return numMatchedTuples;
 }
 
-uint8_t** JoinHashTable::findHashSlot(uint8_t* tuple) const {
-    auto idx = 0u;
-    hash_t hash;
-    entryHashFunctions[idx++](tuple, hash);
-    hash_t tmpHash;
-    while (idx < keyTypes.size()) {
-        entryHashFunctions[idx](tuple + tableSchema->getColOffset(idx), tmpHash);
-        function::CombineHash::operation(hash, tmpHash, hash);
-        idx++;
-    }
+uint8_t** JoinHashTable::findHashSlot(const uint8_t* tuple) const {
+    auto hash = *(hash_t*)(tuple + tableSchema->getColOffset(tableSchema->getNumColumns() - 2));
```

Comment:
we are using `2` in at least two places already. I would make it a const in JoinHashTable.